api_docs/construct-narrow.md

```diff
@@ -34,6 +28,7 @@ as a list of simple objects, as follows:
         "operand": "cool sunglasses"
     }
 ]
+
```

Comment:
Is this line intentional?

Reply:
It was not intentional. I'll get rid of it